lambdas/llama-2-13b-chat-llm/template.yml

```diff
@@ -0,0 +1,112 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: QnABot on AWS LLM Plugin for Llama-2-13b-chat
+
+Parameters:
+
```

Comment:
I think we need parameters to identify the (existing)
- SageMakerEndpointName, and 
- SageMakerEndpointArn

And add more helpful text to the parameter descriptions to make it very clear how they should provision the endpoint (eg Jumpstart docs) and obtain the Name and Arn.

Reply:
Addressed in the 2nd commit.